protocol-definitions/custom/Custom.yaml

```
@@ -1499,3 +1499,15 @@ customTypes:
         type: String
         since: 2.7
         nullable: true
+  - name: Version
+    since: 2.7
```

```suggestion
    since: 2.8
```
For 5.5 release, we need to increment the version.

---

74593a07